### DISCUSSION TITLE
Issue in 1.9.3 Cocoapod CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER .

### DISCUSSION THREAD

**Discussion by SwapnilAIT** [Homebrew/discussions#64] [Everyday usage]
Hi team , I have been using cocoa Pod using homebrew , lates version v 1.9.3 .
Not after update is Firebase , cocoa pod is creating problem with "" and <> in Firebase SDK.
This is creating problem in CICD .
Need help to fix this .
Anyone able to fix this error , please share the knowledge , it will he highly appreciated .

**Comment by SwapnilAIT** [comment]
❌  /Users/vagrant/git/Pods/Target Support Files/nanopb/nanopb-umbrella.h:13:9: double-quoted include "pb.h" in framework header, expected angle-bracketed instead [-Werror,-Wquoted-include-in-framework-header]
#import "pb.h"
                    ^~~~~~~~~~~~~~
❌  /Users/vagrant/git/Pods/Target Support Files/nanopb/nanopb-umbrella.h:14:9: double-quoted include "pb_common.h" in framework header, expected angle-bracketed instead [-Werror,-Wquoted-include-in-framework-header]
#import "pb_common.h"
        ^~~~~~
❌  /Users/vagrant/git/Pods/nanopb/pb_common.h:8:10: double-quoted include "pb.h" in framework header, expected angle-bracketed instead [-Werror,-Wquoted-include-in-framework-header]
#include "pb.h"
        ^~~~~~~~~~~~~
❌  /Users/vagrant/git/Pods/Target Support Files/nanopb/nanopb-umbrella.h:15:9: double-quoted include "pb_decode.h" in framework header, expected angle-bracketed instead [-Werror,-Wquoted-include-in-framework-header]
#import "pb_decode.h"
         ^~~~~~
❌  /Users/vagrant/git/Pods/nanopb/pb_decode.h:9:10: double-quoted include "pb.h" in framework header, expected angle-bracketed instead [-Werror,-Wquoted-include-in-framework-header]
#include "pb.h"
        ^~~~~~~~~~~~~
❌  /Users/vagrant/git/Pods/Target Support Files/nanopb/nanopb-umbrella.h:16:9: double-quoted include "pb_encode.h" in framework header, expected angle-bracketed instead [-Werror,-Wquoted-include-in-framework-header]
#import "pb_encode.h"
         ^~~~~~
❌  /Users/vagrant/git/Pods/nanopb/pb_encode.h:9:10: double-quoted include "pb.h" in framework header, expected angle-bracketed instead [-Werror,-Wquoted-include-in-framework-header]
#include "pb.h"
        ^~~~~~~~~~~~~
❌  /Users/vagrant/git/Pods/Target Support Files/nanopb/nanopb-umbrella.h:17:9: double-quoted include "pb.h" in framework header, expected angle-bracketed instead [-Werror,-Wquoted-include-in-framework-header]
#import "pb.h"
         ^~~~~~
❌  /Users/vagrant/git/Pods/Target Support Files/nanopb/nanopb-umbrella.h:18:9: double-quoted include "pb_decode.h" in framework header, expected angle-bracketed instead [-Werror,-Wquoted-include-in-framework-header]
#import "pb_decode.h"
        ^~~~~~
❌  /Users/vagrant/git/Pods/Target Support Files/nanopb/nanopb-umbrella.h:19:9: double-quoted include "pb_common.h" in framework header, expected angle-bracketed instead [-Werror,-Wquoted-include-in-framework-header]
#import "pb_common.h"
        ^~~~~~~~~~~~~
❌  /Users/vagrant/git/Pods/Target Support Files/nanopb/nanopb-umbrella.h:20:9: double-quoted include "pb.h" in framework header, expected angle-bracketed instead [-Werror,-Wquoted-include-in-framework-header]
#import "pb.h"
        ^~~~~~~~~~~~~
❌  /Users/vagrant/git/Pods/Target Support Files/nanopb/nanopb-umbrella.h:21:9: double-quoted include "pb_encode.h" in framework header, expected angle-bracketed instead [-Werror,-Wquoted-include-in-framework-header]
#import "pb_encode.h"
        ^~~~~~
❌  /Users/vagrant/git/Pods/Target Support Files/nanopb/nanopb-umbrella.h:22:9: double-quoted include "pb_common.h" in framework header, expected angle-bracketed instead [-Werror,-Wquoted-include-in-framework-header]
#import "pb_common.h"
        ^~~~~~~~~~~~~
❌  /Users/vagrant/git/Pods/GoogleDataTransportCCTSupport/GoogleDataTransportCCTSupport/GDTCCTLibrary/Protogen/nanopb/cct.nanopb.h:22:10: could not build module 'nanopb'
#include <nanopb/pb.h>
        ^~~~~~~~~~~~~

**Reply by SMillerDev** [reply]
This is a known issue in cocoapods. Nothing we can do about this.
